src/pal/src/cruntime/misctls.cpp

```diff
@@ -128,6 +128,20 @@ PAL_ctime( const PAL_time_t *clock )
     return retval;
 }
 
+unsigned int GetExponent(double d)
+{
+    return (*((unsigned int*)&d + 1) >> 20) & 0x000007ff;
+}
+
+unsigned long long GetMantissa(double d)
```

Comment:
I'll change that.

---
In reply to: [108800801](https://github.com/dotnet/coreclr/pull/10572#discussion_r108800801) [](ancestors = 108800801)